portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,188 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav class="navbar navbar-expand-sm bg-light navbar-light sticky-top" id="nav">
+        <ul class="navbar-nav">
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#gettoknowme">HOME</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#projectshowcase">Project Showcase</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#kodakmoments">Kodak Moments</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#cameraeatsfirst">Camera Eats First</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" style="color:black" href="#step,brush,tap!">Step, Brush, Tap!</a>
+            </li>
+        </ul>
+    </nav>
+    <!--
+    <div class="intro-image">
+        <img src="images/just-me.jpg" style="height:500px;width:1480px;background-repeat:no-repeat;background-position:center;height:100%" onclick="this.style.display='none'">
     </div>
+    -->
+    <div class="header">
+        <div class="progress-container">
+            <div class="progress-bar" id="myBar"></div>
+        </div>
+    </div>
+    <div class="box">
+        &nbsp;
```

Comment:
It's generally preferable to control formatting with CSS than rely on `&nbsp;` because you can adapt better to varying viewport sizes. It might be worth experimenting with [CSS grid layout](https://gridbyexample.com/examples/) in a future CL.

Reply:
I do have been struggling with space a little bit, and I discovered the "&nbsp" element by searching for options. I do think your point is very important. I will take a closer look at that resource for future projects.